Repository/AbstractPhpcrRepository.php

```diff
@@ -147,6 +148,15 @@ public function add($path, $resource)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function reorder($sourcePath, $position)
+    {
+        Assert::greaterThanEq($position, 0, 'Reorder position cannot be negative, got: %s');
+        $this->reorderNode($sourcePath, $position);
```

Comment:
being a protected method, somebody could still extend the abstract
repository and do custom stuff, so changing the name once its released
is a BC break.

but i can live with having Node in the name, especially as its
consistent with the rest of the class. (that the odm uses nodes
underneath is no argument imho, its behind an abstraction layer.)